README.md

```diff
@@ -119,11 +119,11 @@ print(result)
 
 The compiled function `jfoo` takes and returns PyTorch tensors, just like the original function, so modules and functions compiled by Thunder can be used as part of larger PyTorch programs.
 
-## Train models
+## Model training with Thunder
```

Comment:
Let's keep `Train models` as the title

Reply:
I don't know where the "Train models" came from. In my copy it said "Running training" hence the change. But I like "Train models" even better than "Model training with models"